nautobot/utilities/templates/utilities/templatetags/filter_form_modal.html

```diff
@@ -0,0 +1,83 @@
+{% load form_helpers %}
+{% load static %}
+
+{% block extra_styles %}
+<style>
+    #filterform-table {
+        table-layout: fixed;
+    }
+    #filterform-table thead th:last-child {
+        width: 5em;
+    }
+</style>
+{% endblock %}
+
+<div class="modal fade" tabindex="-1" id="{{ filter_form_name }}_modal">
+    <div class="modal-dialog modal-lg" style="width: 90% !important;">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Filter Form</h4>
+            </div>
+            <div class="modal-body">
+                <ul id="tabs" class="nav nav-tabs">
+                    <li role="presentation" class="active">
+                        <a href="#default-filter" role="tab" data-toggle="tab">
+                            Default
+                        </a>
+                    </li>
+                    <li role="presentation" class="">
+                        <a href="#advanced-filter" role="tab" data-toggle="tab">
+                            Advanced
+                        </a>
+                    </li>
+                </ul>
+                <div class="tab-content">
+                    <div id="default-filter" role="tabpanel" class="tab-pane active">
+                        {% if filter_form %}
+                            {% include 'inc/search_panel.html' %}
+                        {% endif %}
+                    </div>
+                    <div id="advanced-filter" role="tabpanel" class="tab-pane">
+                        {% if dynamic_filter_form %}
```

Comment:
Should this ever not be true? If that's a possibility, then again I'd rather that we not render the tab pane at all rather than rendering an empty tab pane.

Reply:
Yr, If no filterset